### ISSUE TITLE
NuGet stealth downgrades package when batch updating its dependencies

### ISSUE THREAD

**Issue by carolynvs** [NuGet/NuGetGallery#2851]
tldr: When batch updating packages, NuGet should not downgrade other packages.

* openstack.net 1.5.0.1 did not set upper bounds on its dependencies.
* openstack.net 1.5.0.2 was released to set an upper bound to deal with the new NuGet ability to get the latest of dependencies.

After installing openstack.net 1.5.0.2, I immediately see 4 available updates (for its dependencies). This isn't ideal behavior as when I attempt to individually update any of these, it will fail with a message in the Output window. It would be better if it did not display the updates, instead of letting me try and fail but at least it doesn't do any harm. :smile: 

The big problem is that when I select all available updates, NuGet decides that instead of failing, it should **downgrade** the openstack.net package. This is bad because it was not what I requested and is unexpected behavior. I intended to update the checked packages to the latest version, not downgrade some other package. If I do not have the "Show preview window" setting enabled, or fail to carefully read it, I end up screwing up my project.

![bad-nuget](https://cloud.githubusercontent.com/assets/1368985/12181169/41862102-b546-11e5-88ae-4273b5c5335a.png)

**Comment by maartenba** [comment]
This issue was moved to NuGet/Home#1903
